database/v2/schemas/public_profile.model.js

```diff
@@ -0,0 +1,41 @@
+module.exports = (sequelize, type) => {
+  const MODEL = sequelize.define(
+    'PublicProfiles',
+    {
+      id: {
+        type: type.INTEGER,
```

Comment:
id type for PublicProfiles is Integer, but this id has a relation with UserProfile  where id is UUID.

Reply:
Fixed